charts/amoro/values.yaml

```diff
@@ -124,8 +124,30 @@ optimizer:
       jobUri: "local:///opt/flink/usrlib/optimizer-job.jar"
     properties: {
       "flink-conf.taskmanager.memory.managed.size": "32mb",
-      "flink-conf.taskmanager.memory.netwrok.max": "32mb",
-      "flink-conf.taskmanager.memory.netwrok.nin": "32mb"
+      "flink-conf.taskmanager.memory.network.max": "32mb",
+      "flink-conf.taskmanager.memory.network.min": "32mb"
+    }
+  # spark optimizer
+  spark:
+    ## enable the spark optimizer.
+    enabled: true
+    ## container name, default is spark
+    name: ~
+    ## image information of spark optimizer.
+    image:
+      ## the image repository
+      repository: arctic163/optimizer-spark
+      ## the image tag, if not set, the default value is the same with amoro image tag.
+      tag: ~
+      jobUri: "local:///opt/spark/usrlib/optimizer-job.jar"
+    properties: {
+      "master": "k8s://https://<k8s-apiserver-host>:<k8s-apiserver-port>",
+      "deploy-mode": "cluster",
+      "export.HADOOP_USER_NAME": "hadoop",
+      "spark-conf.spark.shuffle.service.enabled": "false",
+      "spark-conf.spark.dynamicAllocation.shuffleTracking.enabled": "true",
```

Comment:
Yes, add the parameter `spark.dynamicAllocation.enabled=true` is useful. Since Spark 3.0, the DRA can run without ESS. The relative feature called Shuffle Tracking was introduced by [SPARK-27963](https://issues.apache.org/jira/browse/SPARK-27963).  If discarding these parameters, we will meet the errors: Caused by: org.apache.spark.SparkException: Dynamic allocation of executors requires the external shuffle service. You may enable this through spark.shuffle.service.enabled.
	at org.apache.spark.ExecutorAllocationManager.validateSettings(ExecutorAllocationManager.scala:213)
	at org.apache.spark.ExecutorAllocationManager.<init>(ExecutorAllocationManager.scala:135)
	at org.apache.spark.SparkContext.<init>(SparkContext.scala:644)
	at org.apache.spark.SparkContext$.getOrCreate(SparkContext.scala:2714)
	at org.apache.spark.sql.SparkSession$Builder.$anonfun$getOrCreate$2(SparkSession.scala:953)
	at scala.Option.getOrElse(Option.scala:189)